src/Purpose.html

```diff
@@ -5563,79 +5563,139 @@
 
 &lt;% } %&gt;
 
-[[[Stand up]|C1P61]]</tw-passagedata><tw-passagedata pid="286" name="C1P61" tags="page" position="3075,15650" size="100,100">&lt;% window.story.delayedText(15 * 1000) %&gt;
+[[[Stand up]|C1P61]]</tw-passagedata><tw-passagedata pid="286" name="C1P61" tags="page chase" position="3075,15650" size="100,100">&lt;ui&gt;standard&lt;/ui&gt;
+&lt;character.one.pain.neutral.neutral-backpack&gt;Sarah&lt;/character&gt;
 
-I get back up and step away from the edge.
 
-Alright, maybe if-
+&lt;action&gt;I get back up and step away from the edge.&lt;/action&gt;
+&lt;action&gt;Alright, maybe if-&lt;/action&gt;
 
 &lt;% if (!window.story.getChoice(&quot;Chapter1&quot;, &quot;StrangerDied&quot;)) { %&gt;
 
-&quot;Shit, watch out!&quot;, the stranger shouts.
+&lt;character.two&gt;Unknown&lt;/character&gt;
+
+&lt;speech.unknown&gt;Shit, watch out!&lt;/speech&gt;
+
+&lt;action&gt;the stranger shouts.&lt;/action&gt;
 
 &lt;% } %&gt;
 
-&quot;Argh!&quot;, I let out.
+&lt;speech.sarah&gt;Argh!&lt;/speech&gt; 
+&lt;action&gt;I let out.&lt;/action&gt;
 
 &lt;% if (window.story.getChoice(&quot;Chapter1&quot;, &quot;Haircut&quot;)) { %&gt;
 
-Something is grabbing my jacket! I spin around to see a walker hand flailing through the window, I instinctively take a few steps back- SHIT, too many steps! I step backward right off the patio roof.
+&lt;characteroverride.two&gt;???&lt;/characteroverride&gt;
+&lt;resetcharacter.two/&gt;
+
+&lt;action&gt;Something is grabbing my jacket!&lt;/action&gt;
+&lt;action&gt;I spin around to see a walker hand flailing through the window, I instinctively take a few steps back- SHIT, too many steps!&lt;/action&gt;
+&lt;action&gt;I step backward right off the patio roof.&lt;/action&gt;
 
 &lt;% } else { %&gt;
 
-Something is grabbing my hair! I get pulled backward, pain shooting over my scalp. I fall down and the grip releases, looking up, I see a walker hand through the window. I roll out of the way and start to get up- uh oh.
+&lt;action&gt;Something is grabbing my hair! &lt;/action&gt;
+&lt;action&gt;I get pulled backward, pain shooting over my scalp. &lt;/action&gt;
+&lt;action&gt;I fall down and the grip releases, looking up, I see a walker hand through the window. &lt;/action&gt;
+&lt;action&gt;I roll out of the way and start to get up- uh oh.&lt;/action&gt;
 
 \* Creeeeeeak, crack \*
 
-My hand, then my arm, then my whole body goes through the patio roof as it collapses inwards.
+&lt;action&gt;My hand, then my arm, then my whole body goes through the patio roof as it collapses inwards. &lt;/action&gt;
 
 &lt;% } %&gt;
 
 &lt;div-#delayed&gt;
 
-I feel the strangest sense of relaxation... the air and rain rushing over me as I rapidly approach the ground. But it only lasts a moment.
+&lt;action&gt;I feel the strangest sense of relaxation... the air and rain rushing over me as I rapidly approach the ground. &lt;/action&gt;
+&lt;action&gt;But it only lasts a moment. &lt;/action&gt;
  
 [[[Hit the ground]|C1P61R1]] 
```

Comment:
Same as previous with `choice` tags.

```suggestion
<choices>[[[Hit the ground]|C1P61R1]] </choices>
```